app/views/shared/org_selectors/_local_only.html.erb

```diff
@@ -36,13 +35,11 @@ placeholder = _("Begin typing to see a list of suggestions.")
     selects or enters something %>
 <% if form.object[id_field]&.to_s =~ /[0-9]+/ || form.object[id_field].nil? %>
   <% val = presenter.crosswalk_entry_from_org_id(value: form.object[id_field]) %>
-  <%= form.hidden_field id_field, value: val, class: "autocomplete-result",
+  <%= form.hidden_field id_field, value: form.object[id_field], class: "autocomplete-result",
                                               autocomplete: "off" %>
 <% else %>
   <%= form.hidden_field :org_id, class: "autocomplete-result",
                                  autocomplete: "off" %>
 <% end %>
 
-<p class="autocomplete-warning red hide">
-  <%= _("The name you entered was not one of the listed suggestions!") %>
-</p>
```

Comment:
I wouldn't remove this warning just now as we might want to shift to the behavior where we do get a drop down list.

Reply:
Although removing it would make it cleaner as we are not using it right now. We could reference the older versions when/if we want to change this behavior.